__tests__/api/request/support.test.ts

```diff
@@ -105,6 +105,9 @@ function runEmailAPITests() {
         supportType: "support",
         name: "name",
         email: "email@email.com",
+        department: "Department",
+        branch: "Branch",
+        title: "Job Title",
```

Comment:
Would it be helpful if we put back the old name/variable but just did it in camel case? 
```suggestion
        jobTitle: "Job Title",
```